klaviyo/api_helper.py

```diff
@@ -212,10 +243,13 @@ def _request(self, method, url, params=None, data=None, request_type=PRIVATE):
                         v1/v2 returns (dict, list).
         """
         self._is_valid_request_option(request_type=request_type)
-        headers = {
-            'Content-Type': 'application/json',
-            'User-Agent': 'Klaviyo-Python/{}'.format(__version__)
-        }
+
+        if headers is None:
```

Comment:
i suggested a refactor, i'd prefer to go that route where headers is an empty dict by default and we have a base constant. then if headers are passed in the constant gets `.update` on it to modify the current headers.

Reply:
just saw it, threading thoughts there

Reply:
Beware of modifying that constant in case it gets used in subsequent requests. You probably want to deepcopy it first